csp-billing-adapter-local.spec

```diff
@@ -19,7 +19,7 @@
 %global skip_python2 1
 %define pythons python3
 Name:           csp-billing-adapter-local
-Version:        0.1.1
+Version:        0.2.0
```

Comment:
Not sure we have been doing bumpversions within PRs like this; will need to remember to manually create the version tag associated with the post-PR-merge commit sha. Check with @smarlowucf

Reply:
Tag is created already ?
No big issue to remove the bumpversion

Reply:
Bumpversion in a PR is fine. So long as there's not other PRs in flight that might cause merge conflicts. We do want the tag created as well.